docs/introduction/zns.md

```diff
@@ -47,91 +47,89 @@ title="Conventional SSDs and ZNS SSDs internal data placement"/>
+starting from the first logical block of each zone. It is always less than or equal
+to the zone size.
 
-This new attribute was introduced to allow for the zone size to remain a power
-of two number of logical blocks (facilitating easy logical block to zone number
-conversions) while allowing optimized mapping of a zone storage capacity to the
-underlying media characteristics. For instance, in the case a flash based
-device, a zone capacity can be aligned to the size of flash erase blocks without
-requiring that the device implements a power-of-two sized erased block.
+This new attribute allows the zone size to remain a power-of-two (2^n) number of
+logical blocks (facilitating easy logical block to zone number conversions) while
+allowing optimized mapping of zone storage capacity to the underlying media
+characteristics. For instance, in the case a flash based device, zone capacity
+can be aligned to the size of flash erase blocks without requiring the device to
+implement a power-of-two sized erase block.
 
 The figure below illustrates the zone capacity concept.
 
 <Image src="intro-zonesize-vs-capacity.png"
 title="Zone Size and Zone Capacity"/>
 
-As the logical block addresses between the zone capacity and the end of the
-zone are not mapped to any physical storage blocks, write accesses to
-these blocks will result in an error. Therefore, reading in this area is handled
-in the same way as when reading unwritted blocks.
+The logical block addresses between the zone capacity(ZCAP) and the end of the zone
+(ZSZE) are not mapped to any physical storage blocks and writes to these blocks
+will result in an error. Therefore, reads within this area are handled
+in the same way as reads to unwritted blocks.
 
-A zone with a zone capacity smaller than the zone size will be transitioned to a
-full condition when the number of written blocks equals the zone capacity.
+A zone with a zone capacity smaller than the zone size will be transitioned to
+zone state Full when the number of written blocks equals the zone capacity.
 
 :::note
 The total namespace capacity reported by a controller is always equal to the
-total number of logical blocks defined by the zones. In other words, this
-reported capacity includes unusable logical blocks of zones with a zone capacity
-lower than the zone size. The usable capacity of the namespace is equal to the
-sum of all zones capacities. This usable capacity is always smaller than the
-reported namespace capacity if the namespace contains zones with a zone capacity
-lower than the zone size.
+total number of logical blocks defined by the zones. In other words, if the
+namespace contains zones with zone capacity less than the zone size, the
+reported capacity includes unusable logical blocks (falling b/w ZCAP & ZSZE) and
+the usable capacity of the namespace is equal to the sum of all zones' capacities.
+In this case, the usable capacity available is smaller than the reported namespace
+capacity.
```

Comment:
if the namespace contains zones with a capacity that is less than their size, the
reported namespace total capacity includes unusable logical blocks (falling b/w ZCAP & ZSZE) and
the usable capacity of the namespace is equal to the sum of all zones' capacities.
In this case, the usable capacity available is smaller than the reported namespace
capacity.

And spell out " b/w ZCAP & ZSZE". No obscure code words please. Make sentences.
-> "between the zone capacity and the zone last block"

Reply:
Updated